src/endpoint/s3/ops/s3_put_bucket_notification.js

```diff
@@ -10,13 +10,37 @@ const notif_util = require('../../../util/notifications_util');
 async function put_bucket_notification(req) {
 
     let topic_configuration = req.body.NotificationConfiguration?.TopicConfiguration;
+    const default_connection = req.object_sdk.requesting_account.default_connection;
 
     //adapt to db shcema
     if (topic_configuration) {
         for (const conf of topic_configuration) {
             conf.id = conf.Id;
             conf.event = conf.Event;
             conf.topic = conf.Topic;
+            //handle Kafka's topic synax, if present
+            if (conf.Topic && conf.Topic.length > 0 && conf.Topic[0].startsWith('kafka:::topic/')) {
+                //kafka_topic_parts[0] = 'kafka:::topic'
```

Comment:
I assume this is the structure of the kafka_topic_parts after the split. maybe its just me but it looks like code that was commented out. can you maybe add a line above that that says something like: `kafka_topic_parts object pasts:`. maybe it will be clearer that way